_includes/templates/download.html

```diff
@@ -86,6 +80,10 @@ <h2>{{ page.latestversion }} {{CURRENT_RELEASE}} <a type="application/rss+xml" h
 
 </div>
 
+  {% if page.version > 2 %}
+  {{page.ubuntu_warning}} <a href="https://bugs.launchpad.net/snapstore/+bug/1803914">1</a>.
```

Comment:
The number "1" by itself looks odd now that there's no longer a number 2.  I suggest converting this into a clickable link icon.  That doesn't require translation, and if we ever get additional links, it'll be easy to convert it back into a number 1.  It'll look like this:

![2019-01-10-12_01_11_545518557](https://user-images.githubusercontent.com/61096/50984273-7aa61000-14cf-11e9-9dcf-a7381b96b079.png)

```diff
diff --git a/_includes/templates/download.html b/_includes/templates/download.html
index d7272b9..75fddd8 100644
--- a/_includes/templates/download.html
+++ b/_includes/templates/download.html
@@ -81,7 +81,7 @@
 </div>
 
   {% if page.version > 2 %}
-  {{page.ubuntu_warning}} <a href="https://bugs.launchpad.net/snapstore/+bug/1803914">1</a>.
+  {{page.ubuntu_warning}} <a class="fa fa-link" href="https://bugs.launchpad.net/snapstore/+bug/1803914"></a>
   {% endif %}
 
   <h2 style="text-align: center">{{ page.patient }}</h2>
```